__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>stop saying "variable for..." in every line's comment. It's obvious that it is a variable lol. The comment should explain WHY, not what</p><p> </p><p>Why have you plotted this? Answer to yourself before reading on. </p><p> </p><p>This plot is useless. No insight is given by knowing the number of the model. This graph makes it look like accuracy is a function of model number, which it is not, because i could equally have shuffled the order of which hyperparams i tried (or even more simply, done the loops in the opposite order).</p><p> </p><p>I think what you mean to plot is a surface plot where height is the accuracy/loss and max_depth varies along the x-axis, and max_features varies along the y-axis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I think I should have presented many small plots instead of this one plot.</p><p>Also, I should have assign sensible axis-description for each sub-plot.</p>